### PR TITLE
Draft of extractor that uses data_processing_api to generate movement modifiers file #700

### PR DESCRIPTION
**Pull request by anthonyohare**
Haven't quite got the hang of the config files needed to upload to the data pipeline.

### CODE REVIEW

File: extractors/aggregate_flows_from_google_data.py [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+    return dfPop, total_population
+
+def download_lookup_table():
+    """
+    Downloads the mapping of iso_3166_2 codes to local authority from the SCRC database, 
+    if it doesn't exist, upload it
+    :return: A dataframe containing the full ISO code (GB-iso_3166_2) and the corresponding local authority code.
+    """
+    
+    #TODO - should I check if these files are in the pipeline and if not upload them before starting?
```
Comment (magicicada) [inline extractors/aggregate_flows_from_google_data.py:91]: For the moment I think the plan is to assume that it is available somewhere as an external object

Comment (anthonyohare) [reply extractors/aggregate_flows_from_google_data.py]: I've tidied up the TODOs and removed the in-script creation of the iso mapping file. If we ever need it we can get it from git.

**Comment by magicicada** [comment]
To make it simpler, I created a new branch off of master after the other extractors were merged, and then brought some of the work from this branch/PR over there, updated it to the newer data pipeline api, and fixed a few other things.  (New PR at https://github.com/ScottishCovidResponse/simple_network_sim/pull/125) Suggest removing this PR, if that's ok with @anthonyohare @bobturneruk ?

**Comment by bobturneruk** [comment]
If the stuff in here is now elsewhere, then yes. Git cherry pick is a neat way to do stuff like this, if you're not already using it.

**Comment by anthonyohare** [comment]
Happy to close it if its not required anymore

**Comment by anthonyohare** [comment]
Already merged in "Amended movement modifiers - issue 700 #125"

**Pull request closed without merging**